### PR TITLE
*: add SeekReverse

### PR DESCRIPTION
**Pull request by disksing**
Not implemented yet, just for saving TiDB from compile error.

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: snapshot.go [hunk 1]
```diff
@@ -214,6 +214,11 @@ func (s *tikvSnapshot) Seek(k kv.Key) (kv.Iterator, error) {
 	return scanner, nil
 }
 
+// SeekReverse creates an reversed Iterator positioned on the first entry which key is less than k.
```
Comment (shenli) [inline snapshot.go]: an -> a

File: txn.go [hunk 2]
```diff
@@ -84,6 +84,12 @@ func (txn *tikvTxn) Seek(k kv.Key) (kv.Iterator, error) {
 	return txn.us.Seek(k)
 }
 
+// SeekReverse creates an reversed Iterator positioned on the first entry which key is less than k.
```
Comment (shenli) [inline txn.go]: an->a

**Comment by shenli** [comment]
LGTM

**Pull request merged**